buildSrc/src/main/kotlin/dev/nolij/zumegradle/JarCompressing.kt

```diff
@@ -57,14 +62,24 @@ fun squishJar(jar: File, classProcessing: ClassShrinkingType, jsonProcessing: Js
 			}
 		}
 	}
-
+	
 	jar.delete()
 	
 	val json = JsonSlurper()
+	
+	val mappings = if (mappingsFile.exists()) mappings(mappingsFile) else null
```

Comment:
Not sure how `null` is a cached value. I more meant to use whatever the Kotlin equivalent of `computeIfAbsent` is...